### ISSUE TITLE
signature of all construct constructors should be "scope, id, props".

### ISSUE THREAD

**Issue by ren-yamanashi** [ren-yamanashi/eslint-cdk-plugin#118]
### Feature Description

Rules to enforce "scope, id, props" or "scope, id" as properties of the constructor of a class that extends Construct.

### Motivation

To unify the Construct structure

### Invalid Code Examples

```typescript
interface MyConstructProps {}

export class MyConstruct extends Construct {
  // ❌ The properties of the constructor must be "scope, id, props" or "scope, id"
  constructor(scope: Construct, id: string, props: MyConstructProps, resoseName: string) { 
  }
}
```

```typescript
interface MyConstructProps {}

export class MyConstruct extends Construct {
  // ❌ The properties of the constructor must be "scope, id, props" or "scope, id"
  constructor(scope: Construct, id: string, myProps: MyConstructProps) { 
  }
}
```

### Valid Code Examples

```typescript
interface MyConstructProps {}

export class MyConstruct extends Construct {
  // ✅ The properties of the constructor is "scope, id"
  constructor(scope: Construct, id: string) { 
  }
}
```

```typescript
interface MyConstructProps {}

export class MyConstruct extends Construct {
  // ✅ The properties of the constructor is "scope, id, props"
  constructor(scope: Construct, id: string, props: MyConstructProps) { 
  }
}
```

### Rule Type

problem (possible errors)

### Implementation Details

- [x] This rule is related to AWS CDK best practices
- [x] I'm willing to submit a PR for this feature

### Additional Context

The same rules apply to awslint.

https://github.com/aws/aws-cdk/blob/0607b2cd48e3ae1cb8c1ab883d9af525c7bb89ff/packages/awslint/lib/rules/construct.ts#L83
